docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
+
+services:
+  zoo1:
+    image: zookeeper:3.4.9
+    hostname: zoo1
+    ports:
+      - "2181:2181"
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_PORT: 2181
+      ZOO_SERVERS: server.1=zoo1:2888:3888
+    volumes:
+      - ./zk-single-kafka-single/zoo1/data:/data
+      - ./zk-single-kafka-single/zoo1/datalog:/datalog
+
+  kafka1:
+    image: confluentinc/cp-kafka:5.3.0
```

Comment:
Out of interest, any specific reason why you've picked confluent over wurstmeister aka Apache Kafka?

Reply:
I've tried both and the wurstmeister image didnt work consistently for me. I dont know why tho'.

Reply:
This is just fine though, was just curious. Might also be that confluent has better documentation, so feel free to stick with that.